src/frontend/komponenter/Fagsak/Forhåndsvarsel/Forhåndsvarsel.test.tsx

```diff
@@ -0,0 +1,100 @@
+import type { DokumentApiHook } from '../../../api/dokument';
+import type { BehandlingHook } from '../../../context/BehandlingContext';
+import type { Behandling } from '../../../typer/behandling';
+import type { Fagsak } from '../../../typer/fagsak';
+import type { RenderResult } from '@testing-library/react';
+import type { NavigateFunction } from 'react-router';
+
+import { render, screen, fireEvent } from '@testing-library/react';
+import React from 'react';
+
+import { Forhåndsvarsel } from './Forhåndsvarsel';
+
+const mockUseBehandling = jest.fn();
+const mockUseDokumentlisting = jest.fn();
+const mockUseDokumentApi = jest.fn();
+
+jest.mock('react-router', () => ({
+    ...jest.requireActual('react-router'),
+    useNavigate: (): NavigateFunction => jest.fn(),
+}));
+
+jest.mock('../../../context/BehandlingContext', () => ({
+    useBehandling: (): BehandlingHook => mockUseBehandling(),
+}));
+jest.mock('../../../api/dokument', () => ({
+    useDokumentApi: (): DokumentApiHook =>
+        mockUseDokumentApi(
+            jest.fn().mockReturnValue({ bestillBrev: jest.fn(), forhåndsvisBrev: jest.fn() })
+        ),
+}));
+jest.mock('../Høyremeny/Dokumentlisting/DokumentlistingContext', () => ({
+    useDokumentlisting: (): DokumentApiHook => mockUseDokumentlisting(),
+}));
+
+const createMockBehandling = {
+    behandlingId: '1',
+    eksternBrukId: '123',
+    varselSendt: false,
+} as unknown as Behandling;
+
+const createMockFagsak = {
+    fagsystem: 'EF',
+    eksternFagsakId: '456',
+} as unknown as Fagsak;
+
+const setupMock = (): void => {
+    mockUseBehandling.mockImplementation(() => ({
+        actionBarStegtekst: jest.fn().mockReturnValue('Steg 1 av 5'),
```

Comment:
Er ikke steget nr.2 i rekken?
```suggestion
        actionBarStegtekst: jest.fn().mockReturnValue('Steg 2 av 5'),
```